compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/FlingBehavior.kt

```diff
@@ -43,7 +43,13 @@ interface FlingBehavior {
      * @return remaining velocity after fling operation has ended
      */
     suspend fun ScrollScope.performFling(initialVelocity: Float): Float
+
+    /**
+     * Update the internal parameters of FlingBehavior in accordance with the new [androidx.compose.ui.unit.Density] value.
+     *
+     * @param density new density value.
+     */
+    fun updateDensity(density: Density) = Unit
 }
 
-@Composable
-internal expect fun rememberFlingBehavior(): FlingBehavior
\ No newline at end of file
+internal expect fun platformDefaultFlingBehavior(): FlingBehavior
```

Comment:
Let's move it to Scrollable.kt, near `internal class DefaultFlingBehavior`, as the current default fling behavior is coupled with Scrollable.

Reply:
Done